src/lib.rs

```diff
@@ -214,17 +199,10 @@
 //! [colors]: colors/index.html
 //! [syslog]: syslog/index.html
 //! [meta]: meta/index.html
-#[cfg(feature = "colored")]
-extern crate colored;
-#[cfg(all(not(windows), feature = "reopen-03"))]
-extern crate libc;
-extern crate log;
 #[cfg(all(not(windows), feature = "syslog-4"))]
-extern crate syslog as syslog_4;
+use ::syslog as syslog_4;
```

Comment:
I don't think `::` here should be necessary

Reply:
I also didn't think this should be necessary, but without the leading `::` you run into this error, which I'd never seen before:

```
error[E0659]: `syslog` is ambiguous (name vs any other name during import resolution)
   --> src/lib.rs:203:5
    |
203 | use syslog as syslog_4;
    |     ^^^^^^ ambiguous name
    |
    = note: `syslog` could refer to an extern crate passed with `--extern`
    = help: use `::syslog` to refer to this extern crate unambiguously
note: `syslog` could also refer to the module defined here
   --> src/lib.rs:227:1
    |
227 | pub mod syslog;
    | ^^^^^^^^^^^^^^^
    = help: use `crate::syslog` to refer to this module unambiguously
```

I could add a comment above, something to the effect of "This external crate import must be qualified with `::` to disambiguate from the internal `syslog` module". How does that sound?

Reply:
Oh, interesting! I guess that makes sense since we do have that module named syslog.

Maybe we could rename the crate in `Cargo.toml` instead?

If we use the `package` key added in 1.31.0 ([doc](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml)), I think that would even let us get rid of the `syslog3` crate entirely. Something like this should work:

```toml
syslog3 = { version = "3", package = "syslog", optional = true }
syslog4 = { version = "4", package = "syslog", optional = true }
```

Reply:
Good idea! Let me know what you think of the changes in de582a0.

Reply:
Realized I never replied to this, sorry!

In any case, these changes look good - and I think this PR is pretty ready given that.